### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by samdark**

### CODE REVIEW

File: src/Command.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 use Psr\Http\Message\ServerRequestInterface;
 
-class Command
+final class Command
```
Comment (samdark) [inline src/Command.php:7]: CURL command syntax is standardized so there should be no need to customize it.

File: src/Command.php [hunk 2]
```diff
@@ -24,22 +24,22 @@ class Command
     /**
      * Single quote character
      */
-    public const QUOTE_CHARACTER_SINGLE = "'";
+    public const QUOTE_SINGLE = "'";
```
Comment (samdark) [inline src/Command.php:27]: `CHARACTER` doesn't add anything to meaning and could be removed.

File: src/Command.php [hunk 3]
```diff
@@ -24,22 +24,22 @@ class Command
...
      * No quote character
      */
-    public const QUOTE_CHARACTER_NONE = '';
+    public const QUOTE_NONE = '';
 
     /**
      * Command name
      */
-    private const NAME = 'curl';
+    private const COMMAND_NAME = 'curl';
```
Comment (samdark) [inline src/Command.php:42]: `NAME` is too generic so you have to look it up.

File: src/Command.php [hunk 4]
```diff
@@ -274,7 +274,7 @@ private function buildTemplatePart($search, $replace): void
      */
     private function getTemplatePartPattern($search): string
     {
-        return '/ ?' . str_replace(['{', '}'], ['\{', '\}'], $search) . ' ?/';
+        return '/ ?' . preg_quote($search, '/') . ' ?/';
```
Comment (samdark) [inline src/Command.php:277]: This in the only serious correction in the whole pull request...

**Comment by alexkart** [comment]
Thanks.

**Pull request merged**